processor/processor.go

```diff
@@ -177,6 +178,7 @@ type Handle struct {
 		userTransformationMirroringFireAndForget  config.ValueLoader[bool]
 		storeSamplerEnabled                       config.ValueLoader[bool]
 		archiveInPreProcess                       bool
+		enableSrcHydrationStage                   bool
```

Comment:
I am not confident that we really need this configuration option. What is the actual risk of having the stage enabled, always? I propose to keep only the necessary configuration options, the ones that matter, i.e. remove this one